testdata/go/v3/memcached-operator/monitoring/metrics/metrics.go

```diff
@@ -0,0 +1,21 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+// MemcachedNumberOfReplicasReconcileCountTotal counts Memcached replicas number reconciliations
```

Comment:
this comment should be above `MemcachedNumberOfReplicasReconcileCountTotal = prometheus.NewCounter(`
and not above `var (`

Reply:
Done